serverless-workflow-examples/serverless-workflow-openvino-quarkus/pom.xml

```
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.kie.kogito.examples</groupId>
+  <artifactId>serverless-workflow-openvino-quarkus</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>Kogito Example :: Serverless Workflow Openvino :: Quarkus</name>
+  <description>Kogito Serverless Workflow OpenVino Example - Quarkus</description>
+  <properties>
+    <quarkus-plugin.version>2.16.6.Final</quarkus-plugin.version>
```

We're on 2.16.7.Final already.
```suggestion
    <quarkus-plugin.version>2.16.7.Final</quarkus-plugin.version>
```